gamestonk_terminal/stocks/discovery/yahoofinance_model.py

```diff
@@ -44,3 +44,32 @@ def get_undervalued() -> pd.DataFrame:
     data = pd.read_html(requests.get(url).text)[0]
     data = data.iloc[:, :-1]
     return data
+
+
+def get_gtech() -> pd.DataFrame:
+    """Get Yahoo Finance growth technology stocks
```

Comment:
Can we have:
"""Get Technology stocks with revenue and earnings growth in excess of 25%. [Source: Yahoo Finance]